resources/aws_ec2.go

```diff
@@ -111,6 +113,36 @@ const (
 	awsEc2EventInstanceExists
 )
 
+// bootstrapScript is a bash script template for installing mgmt and connecting
+// back to the seed specified in the definition.
+const bootstrapScript string = `#!/bin/bash
+if [ -n "$(command -v apt)" ]
+then
+  apt update
+  apt install -y libaugeas0 libvirt0
+fi
+
+if [ -n "$(command -v yum)" ]
+then
+  yum install -y augeas-libs libvirt-libs
+fi
+
+mkdir -p "/opt/mgmtconfig"
```

Comment:
maybe we can use `mktemp` instead of /opt/ ?

Reply:
`mktemp -p "/tmp/mgmtconfig"`?

Reply:
I'll play with it.

Reply:
mktemp -d will get you a temp dir instead of assuming that opt is writable or available.